src/main/java/org/elasticsearch/index/translog/fs/FsTranslog.java

```diff
@@ -237,7 +236,7 @@ public void newTranslog(long id) throws TranslogException, IOException {
                 if (currentFree < size) {
                     size = currentFree;
                     location = file;
-                } else if (currentFree == size && ThreadLocalRandom.current().nextBoolean()) {
+                } else if (currentFree == size) {
```

Comment:
Do we need the else condition at all now? Could we just keep the first if block (meaning we take the first location instead of the last if they are all equal?)